Server/JokesOnYou.Web.Api/Profiles/TagProfile.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AutoMapper;
+using JokesOnYou.Web.Api.DTOs;
+using JokesOnYou.Web.Api.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JokesOnYou.Web.Api.Profiles
+{
+    public class TagProfile : Profile
+    {
+        public TagProfile()
+        {
+            CreateMap<Tag, TagReplyDto>();
+            CreateMap<Joke, JokeReplyDto>();
+            CreateMap<User, UserReplyDTO>();
```

Comment:
Asking to learn here... These maps are already defined in JokesProfile. Won't they overlap with each other? How does AutoMapper know which one you want to use?

Reply:
Yes i did see that. But wouldnt that make this endpoint dependent on that profile?

Reply:
Yeah, I don't know how to do this actually. Normally I would just put all mappings in one big file. But I have no idea if that is the way forward.

Reply:
To follow SRP we have a profile for every entity no need to register the same mapping twice. When you register automapper in DI i bet it uses reflection to find all profiles.